tests/push/test_http.py

```diff
@@ -43,7 +43,6 @@ class HTTPPusherTests(HomeserverTestCase):
 
     def default_config(self) -> Dict[str, Any]:
         config = super().default_config()
-        config["start_pushers"] = True
```

Comment:
Since the main process has it enabled by default?

If so, let's remove the entire `default_config` method.

Reply:
I wasn't sure that was how that works. Now I am.

Reply:
Aaannnnnd, it's gone!